bin/metababel

```diff
@@ -144,7 +142,12 @@ def wrote_event_dispatchers(folder, trace_model, trace_matching_model)
     # Modify the dispatcher to add the new dispatchType
     name_sanitized = sanitize(em.callback_name)
     dispatch_type_args = []
-    dispatch_type_args += [GeneratedArg.new('int64_t', '_timestamp')] if default_clock_class and default_clock_class.fetch(:extract, true)
```

Comment:
I prefer the previous version (less lines), although I can live with the new one.